### PR TITLE
Religions update

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[content]
Did stuff!
:cl:
 * rscadd: The narcissism religion now accepts the adept's name as a key.
 * rscadd: New religions, plus various updates to all religious accepted keys and other misc things.
 * tweak: Religion keys relating to already deity names have been removed, to stop redundancy.
 * bugfix: Observers and silicons no longer answer to Animal Jesus.

### CODE REVIEW

File: code/datums/religions.dm [hunk 1]
```diff
@@ -409,7 +411,7 @@
 	bible_name = "The Satanic Bible" //What I found on Google, ergo the truth
 	male_adept = "Magister"
 	female_adept = "Magistera"
-	keys = list("satan", "evil", "satanism")
+	keys = list("satan", "evil", "satanism", "bad")
```
Comment (jknpj) [inline code/datums/religions.dm]: implying bad

File: code/datums/religions.dm [hunk 2]
```diff
@@ -467,7 +469,7 @@
 	bible_names = list("The Vedas", "The Mahabharata")
 	male_adept = "Guru"
 	female_adept = "Guru"
-	keys = list("hindu", "hinduism", "india")
+	keys = list("hindu", "hinduism", "india", "designated")
```
Comment (Durgen) [inline code/datums/religions.dm]: kek

Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: oh i was gonna go all in but i dont know if either github or people here are fully ok with it

Comment (Durgen) [reply code/datums/religions.dm]: your good bro

File: code/datums/religions.dm [hunk 3]
```diff
@@ -969,7 +969,16 @@
 	bible_type = /obj/item/weapon/storage/bible/booze
 	male_adept = "Komrade"
 	female_adept = "Komrade"
-	keys = list("communism", "socialism", "commie", "russia", "mother russia", "motherland")
+	keys = list("communism", "socialism", "commie")
+
+/datum/religion/marxistleninism
+	name = "Marxist-Leninism"
```
Comment (Durgen) [inline code/datums/religions.dm]: "marxism-leninism" would be more correct than "marxist-leninism"

File: code/datums/religions.dm (hunk 2 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: dun

File: code/datums/religions.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: huh??????? huh??????????????????????

**Comment by SonixApache** [comment]
I'm guessing there's no need to put deity names in the keys because they already get checked?

**Comment by SECBATON-GRIFFON** [comment]
> I'm guessing there's no need to put deity names in the keys because they already get checked?

my point exactly

**Comment by Durgen** [comment]
marxism-leninism would be more correct than marxist-leninism

File: code/datums/religions.dm (hunk 3 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: fixd

**Comment by Eneocho** [comment]
We need more religions that use the craving inducing Incense.

**Comment by SECBATON-GRIFFON** [comment]
> We need more religions that use the craving inducing Incense.

ok, just added one now

**Comment by Kurfursten** [comment]
Most of these new religion keys are really, really low effort garbage and some of them are outright unacceptable (the ones that reference specific player ckeys like "shapka" - we have made player references before but it never says their ckey, e.g.: the toy snowflake). Moreover this is an atomicity nightmare. No one would object to adding the narcissism thing surely, but there are tons of unrelated changes. Some of them are remarkably poorly though through, like giving the chaplain 10 space drugs that will just wear off.

All of this just looks terribly low effort - how about you focus in on just one or a couple religions and really polish them up with some special content instead of just vomiting memes all over the religion keys?

**Comment by SECBATON-GRIFFON** [comment]
ok, some of the worst offender stuff has been removed. plus some like the speedrunning one already had a few meme keys so im not sure where the line is drawn on this.

File: code/datums/religions.dm [hunk 4]
```diff
@@ -1492,26 +1531,174 @@
...
+	female_adept = "Cornholio"
+	preferred_incense = /obj/item/weapon/storage/fancy/incensebox/novaflowers
+	keys = list("cornholio", "tp", "toilet paper", "bunghole", "beavis")
+
+//TODO: some conversion ceremony with surgical drills to make stigmatas from lake titticaca
+/datum/religion/cornholio/equip_chaplain(var/mob/living/carbon/human/H)
+	H.reagents.add_reagent(COFFEE,10) // settle down, beavis
+	H.reagents.add_reagent(SUGAR,10)
+
+/datum/religion/schizo
```
Comment (Kammerjunk) [inline code/datums/religions.dm:1582]: threadman is right, this religion is dumb

File: code/datums/religions.dm [hunk 5]
```diff
@@ -1492,26 +1531,174 @@
...
+	H.reagents.add_reagent(SPACE_DRUGS,10)
+
+/datum/religion/pomf
+	name = "Pomfism"
+	deity_name = "Pomf"
+	bible_name = "The Book of Pomf"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Chicken Farmer"
+	female_adept = "Chicken Farmer"
+	keys = list("pomf", "pomf123", "chicken", "chikun", "pomf chicken", "d3athrow", "badclown", "/vg/station", "/vg/station13", "pomfcoin")
```
Comment (Kammerjunk) [inline code/datums/religions.dm]: so are some of these. like, ckeys? actually just dropping the server name? cmon

Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: pomfs is apparently an allowed exception

File: code/datums/religions.dm (hunk 4 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm:1582]: on a concept or execution level

**Comment by SECBATON-GRIFFON** [comment]
okay, the giving space drugs stuff has been replaced with activating hallucination gene blocks that dont wear off

File: code/datums/religions.dm [hunk 6]
```diff
@@ -1179,7 +1222,7 @@
 	bible_name =  "Honkmothers Coloring Book"
 	male_adept = "Co-Clown"
 	female_adept = "Co-Clown"
-	keys = list("honk", "clown", "honkmother")
+	keys = list("honk", "clown", "shoe snatchin willy", "slip", "banana", "bananas", "slipping", "shoe snatching")
```
Comment (Eneocho) [inline code/datums/religions.dm:1229]: Why did you remove "honkmother" from the list? Also, add aunt scootaloo.

File: code/datums/religions.dm [hunk 7]
```diff
@@ -409,23 +433,23 @@
...
+	keys = list("evil", "satanism", "bad")
 	preferred_incense = /obj/item/weapon/storage/fancy/incensebox/moonflowers
 	bookstyle = "Burning"
 
 /datum/religion/lovecraft
 	name = "Esoteric order of Dagon"
 	deity_name = "Cthulhu" //I hope it's spelt correctly
 	bible_names = list("The Necronomicon", "The Book of Eibon", "De Vermis Mysteriis", "Unaussprechlichen Kulten")
-	keys = list("cthulhu", "old ones", "great old ones", "outer gods", "elder gods", "esoteric order of dagon")
+	keys = list("old ones", "great old ones", "outer gods", "elder gods", "esoteric order of dagon")
```
Comment (Eneocho) [inline code/datums/religions.dm]: Also here, why remove cthulhu.

File: code/datums/religions.dm [hunk 8]
```diff
@@ -409,23 +433,23 @@
...
 	symbolstyle = 5
 	bookstyle = "Necronomicon" //also "Unaussprechlichen Kulten" "Ithaqua"
 
 /datum/religion/hastur
 	name = "Brotherhood of The Yellow Sign" //I'm fed up with people think I worship Dagon. We're moving out.
 	deity_name = "Hastur"
 	bible_name = "The King in Yellow" //The name of the titular fictional play in the 1895 book by Robert Chambers
-	keys = list("hastur","yellow sign","king in yellow","brotherhood of the yellow sign")
+	keys = list("yellow sign","king in yellow","brotherhood of the yellow sign")
```
Comment (Eneocho) [inline code/datums/religions.dm]: or here.

File: code/datums/religions.dm [hunk 9]
```diff
@@ -409,23 +433,23 @@
 	bible_name = "The Satanic Bible" //What I found on Google, ergo the truth
 	male_adept = "Magister"
 	female_adept = "Magistera"
-	keys = list("satan", "evil", "satanism")
+	keys = list("evil", "satanism", "bad")
```
Comment (Eneocho) [inline code/datums/religions.dm]: Or here, why remove satan as a keyword.

**Comment by Eneocho** [comment]
Also, you can use:
	if (H.reagents.get_reagent_amount(reagent) < 5)
		H.reagents.add_reagent(reagent, 5)
to give the player a constant feed of chems. It's how the helpful virology symptoms work (such as space adaptation or hyperzine).
See: https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/modules/virus2/effect/stage_3.dm and related for some examples.

File: code/datums/religions.dm (hunk 6 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm:1229]: because its already a deity name

File: code/datums/religions.dm (hunk 7 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: deity name

File: code/datums/religions.dm (hunk 8 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: ditto

File: code/datums/religions.dm (hunk 9 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm]: ahem

File: code/datums/religions.dm (hunk 4 above)
Comment (Kammerjunk) [reply code/datums/religions.dm:1582]: execution, I guess. the whole "thread = schizos" thing is mad OOC, even worse than the trialmin religion
but those are gone now, I see

File: code/datums/religions.dm [hunk 10]
```diff
@@ -1492,26 +1535,186 @@
...
+
+	convert(subject, preacher)
+	return TRUE
+
+/datum/religion/fightclub
+	name = "Fight Club"
+	deity_name = "Tyler Durden"
+	bible_name = "The Rules of Fight Club"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Tyler Durden" // spoilers?
```
Comment (Eneocho) [inline code/datums/religions.dm:1679]: His name is Robert Paulsen.

**Comment by Eneocho** [comment]
Also, you could add John McAfee as a deity for ancap.

**Comment by ghost** [comment]
i do like the cornholio one, at least

**Comment by BomberBro** [comment]
Some of these are cool, some of the choices like removing "hastur" from the list of words that you put in to get the hastur religion just seem silly and unnecessarily confusing. Also the schizo religion will be dated as fuck in a year at best, its effects should be lumped with the conspiracy theorist one

**Comment by Eneocho** [comment]
> 
> 
> Some of these are cool, some of the choices like removing "hastur" from the list of words that you put in to get the hastur religion just seem silly and unnecessarily confusing. Also the schizo religion will be dated as fuck in a year at best, its effects should be lumped with the conspiracy theorist one

He didn't remove em actually. Apparently inserting a deity name when choosing religion also gives you the deity name's religion, so having Hastur both as a deity and as a keyword would be redundant, since it will give you the hastur religion anyways.

@SECBATON-GRIFFON You should really put that detail on the OP or more people will say the same.

File: code/datums/religions.dm [hunk 11]
```diff
@@ -1492,26 +1539,186 @@
...
+	female_adept = "Tyler Durden"
```
Comment (Eneocho) [inline code/datums/religions.dm:1684]:
```suggestion
	male_adept = "Robert Paulson" // spoilers?
	female_adept = "Robert Paulson"
```

**Review by Eneocho — CHANGES_REQUESTED** [review]

File: code/datums/religions.dm (hunk 11 above)
Comment (SECBATON-GRIFFON) [reply code/datums/religions.dm:1684]: isnt he the guy who dies? makes no sense why itd be that

**Comment by SECBATON-GRIFFON** [comment]
yeah @BomberBro i removed hastur cause it already checks for deity names

File: code/datums/religions.dm (hunk 11 above)
Comment (Eneocho) [reply code/datums/religions.dm:1684]: It makes total sense, that's the pivoting point for the movie. That's when Project Mayhem evolves from a dissident group into an outright cult. In death, a member of Project Mayhem HAS a name. His name is Robert Paulson.

**Comment by Eneocho** [comment]
You should use the permanent reagents system that virology uses for the obesity, cornholio and schizo religions, since a singular genecall will be overwritten by ryetalyn or genetics. Most of the chaplains will at one point or another consume ryetalyn, thus removing the genes and erasing the one thing making those religions special.

**Comment by Kurfursten** [comment]
A gene is fine. Rye will just inactivate the block, so it's easy to get the disability back if you talk to genetics. Clowns put up with this too.

File: code/datums/religions.dm (hunk 11 above)
Comment (Eneocho) [reply code/datums/religions.dm:1684]: Please man, make this change.

**Comment by SonixApache** [comment]
why is this still open?

**Comment by DamianX** [comment]
I'm killing the competition

**Comment by SonixApache** [comment]
✝️

**Comment by Eneocho** [comment]
DORIME
